gen/flights/flightplan.py

```diff
@@ -114,6 +116,51 @@ def tot_waypoint(self) -> Optional[FlightWaypoint]:
         failed to generate. Nevertheless, we have to defend against it.
         """
         raise NotImplementedError
+    
+    @cached_property
+    def bingo_fuel(self) -> int:
+        """Bingo fuel value for the FlightPlan
+        """
+        assert self.farthest_wpt_from_arrival is not None
+        distanceToArrival = meter_to_nm(self.farthest_wpt_from_arrival)
```

Comment:
I'd missed this earlier: `self.flight.arrival` is never `None`. This can just be:

```
distance_to_arrival = self.max_distance_from(self.flight.arrival)
```